NameLayerMain/src/vg/civcraft/mc/namelayer/gui/GUIGroupOverview.java

```diff
@@ -78,14 +77,14 @@ public void clicked(Player p) {
 					NameLayerPlugin.log(Level.INFO,
 							p.getName() + " turned autoaccept for invites off "
 									+ "via gui");
-					NameLayerPlugin.getGroupManagerDao().removeAutoAcceptGroup(p.getUniqueId());
+					NameLayerPlugin.getGroupManagerDao().removeAutoAcceptGroupAsync(p.getUniqueId());
```

Comment:
By directly accessing the DAO here, instead of going through the GroupManager, no mercury message is sent. So, if they are on other shards, those shards won't know that this person now auto accepts.